apps/builder/app/builder/features/style-panel/sections/transforms/transform-utils.ts

```diff
@@ -127,14 +134,22 @@ export const addDefaultsForTransormSection = (props: {
 
 export const isTransformPanelPropertyUsed = (params: {
   currentStyle: StyleInfo;
-  panel: TransformPanel;
+  panel: (typeof transformPanelDropdown)[number];
 }): boolean => {
   const { currentStyle, panel } = params;
   switch (panel) {
     case "scale":
     case "translate":
       return currentStyle[panel]?.value.type === "tuple";
 
+    /*
+      backface-visibility takes only two values, either `hidden` / `visible`
+      And it's not inherited. So, we need to check with the local value.
+      If we check with the computed value, it will always return true.
```

Comment:
comment seems outdated as its not just the 2 values, not sure if the check is outdated too

Reply:
Updated the comment with some more details 👍